### ISSUE TITLE
Paper

### ISSUE THREAD

**Issue by Seaor** [Seaor/SISR#1]
To LinTai

**Comment by Seaor** [comment]
Training details
第一段不变
All experiments are performed with 544x544 images. The mini-batch size is set to 32. We observe that model results benfits from multi-scale training, since multi-scale training helps the model to adapt to frequent change of scale fatures. Specifically, we select different scales for training under different iterations.
The while training process is divided into two stages. First, we train the backbone network on the ImageNet dataset. Pre-training reduces training time and helps the CPDCNN to easier converges. We then employ th pre-trained model as an initialization to further fine-tunning for the people detection task. The model is trained using the localization loss function in Formula (6) with $\alpha = 0.5$ and $\bata = 0.5$. The learning rate is set to 0.0001 and adjusted by the multi-stepping strategy.
We use three sub-networks for our CPDCNN to resolve multi-scale detection problem, three sub-networks are weight-sharing and each of them contains 4 RDB blocks. The dilated rate is set to 1,2 and 3 respectively. We implement our model with the tensorflow framework and train them on 2 Tesla K80s with 80000 iterations.
Ablation Study
In order to better understand the effects of each component in proposed CPDCNN, we gradually modify the baseline YOLOv3 model and compare their results. Table2 shows our proposed CPDCNN achieves 5 points performance gain compared to the baseline model with similar settings.
We first replace feature pyramid network by three parallel sub-networks containing our proposed RDB blocks for multi-scale detection problem. It helps capture the features of multi-scale pedestrian instances and save the computational costs. Moreover, we use GIoU and repulsion loss items as our localization loss to eliminate the influences of surrounding objects in the crowd. We also use some training strategies to speed up training and get rid of over-fitting.
Thanks to the improvements mentioned above, our CPDCNN model shows its superior performance compared with other detection-based people counting methods. A specific discussion is provided as follows.
Experimental Results
We compare our proposed CPDCNN on several public benchmark datasets with state-of-the-art detection-based people counting methods. It can be observed from Table 3 that our proposed method outperforms previous methods in both average precision and running speed. The highlighted sign indicates the best result compared with the methods in the table. It is difficult to directly compare the running speed results of different detectors, since the speed varies with experimental conditions. Thus we report the speed results of all methods achieved on the same computer with a GTX 1080Ti. In the same experimental environment, the accuracy of our method is significantly higher than those of other algorithms and still remains its real-time performance.
We also evaluate our method on the CrowdHuman dataset. In comparison on existing datasets, the CrowdHuman dataset contain larger and more scens of crowds. Statistics show that CityPersons dataset has approximately six persons per image, the Caletech-USA and KITTI datasets have fewer persons per image. In these datasets, there are few scenes of crowds. So the CrowdHuman dataset is well-suited for our study to verify the performance of the detector in highly crowd scenes. We use the CrowdHUman validation set as the test set, and compare the results of our CPDCNN and the current state-of-the-art one-stage methods, such as SSD and RetinaNet. The results shown in Table 4 indicate that our method is still robust to human detection in dense scenes.
In addition 关于Seq-NMS部分不变。
